src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Use Roboto font */
+body {
+  font-family: Roboto, sans-serif;
+}
+
+/* Header styling */
+header {
```

Comment:
Consider using classes for styling

Reply:
Not fixed:

![image](https://github.com/user-attachments/assets/e673b7e5-ee0f-4a02-82b7-bb0e52a08944)

Fix all similar cases

Reply:
Show an example of how I should do it. I have everything divided by classes